datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -536,7 +536,7 @@ def database_monitoring_query_metrics(self, raw_event):
         aggregator.submit_event_platform_event(self, self.check_id, to_native_string(raw_event), "dbm-metrics")
 
     def _submit_metric(
-        self, mtype, name, value, tags=None, hostname=None, device_name=None, raw=False, flush_first_value=False
+        self, mtype, name, value, tags=None, hostname=None, device_name=None, raw=False, flush_first_value=False,
```

Comment:
I think this change is failing CI due to style, probably better just to not include it.

Reply:
Yeah, seems a mistake to include that change